src/components/users/ReferralSendForm.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from "react";
+import { Formik, Form } from "formik";
+import Input from "components/forms/Input";
+import ErrorMessage from "components/ErrorMessage";
+import SubmitSpinnerButton from "components/forms/SubmitSpinnerButton";
+import isEmail from "validator/lib/isEmail";
+import { AuthContext } from "components/auth/Auth";
+
+const ReferralSendForm = ({ onSubmit, submitText = "Send Referral" }) => {
+  const { user } = useContext(AuthContext);
+  const initialValues = {
+    email: "",
+    referralLink: `${process.env.REACT_APP_URL}/register?referralCode=${user.id}`
```

Comment:
This environment variable is not documented anywhere, nor is it in the `example.env` file. It's causing the referral link to insert `undefined` instead of the app's url.

Reply:
![Screenshot_2021-01-08 Get Paid In Bitcoin](https://user-images.githubusercontent.com/26264269/103955890-11773700-5193-11eb-8e64-7d54cef4ce3b.png)